recipes-soc/displayinfo-soc/displayinfo-soc_git.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "DisplayInfo SOC specific code for RaspberryPi"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://../LICENSE;md5=5bfad6e034e497ee148eec56e175c6e8"
+
+inherit cmake pkgconfig
+
+PROVIDES += "virtual/vendor-displayinfo-soc"
+RPROVIDES:${PN} = "virtual/vendor-displayinfo-soc"
+
+DEPENDS += "libdrm"
+RDEPENDS:${PN} += "libdrm"
+
+SRC_URI = "${CMF_GITHUB_ROOT}/rdkvhal-rdkservices-raspberrypi4;${CMF_GITHUB_SRC_URI_SUFFIX}"
+
+S = "${WORKDIR}/git/DisplayInfo"
+
+export LINUX_BUILD="1"
+
+FILES_SOLIBSDEV = ""
+SOLIBS = ".so"
+INSANE_SKIP:${PN} += "dev-so"
+
+ALLOW_EMPTY:${PN} = "1"
+
+FILES:${PN} += "${libdir}/libdisplayinfo-soc.so"
```

Comment:
Is the MW mandating a specific name for the library?

Reply:
yes, MW expects displayinfo-soc as the library name